packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -567,6 +567,7 @@ class MemoryController extends DisposableController
     _memoryTracker.start();
 
     // Log Flutter extension events.
+    // TODO(kenz): do we want to listen with event history here?
```

Comment:
@terrylucas - any input on whether we should listen to event history here?

Reply:
unless we were able to get historic events about total memory usage there is probably no need to.